src/components/shared/ExerciseSide.tsx

```diff
@@ -1,7 +1,40 @@
 import '../../styles/ExerciseSide.scss';
+import { UnitCircleTick, UnitCircleExercise } from './UnitCircleExercise';
+
+const labelOne: UnitCircleTick = {
+  label: 'A',
+  type: 'unit-circle-unknown-label',
+};
+const labelTwo: UnitCircleTick = {
+  label: 'C',
+  type: 'unit-circle-unknown-label',
+};
+const labelThree: UnitCircleTick = {
+  label: '0˚',
+  type: 'unit-circle-known-label',
+};
+const labelFour: UnitCircleTick = {
+  label: 'B',
+  type: 'unit-circle-unknown-label',
+};
+const labelFive: UnitCircleTick = {
+  label: '90˚',
+  type: 'unit-circle-known-label',
+};
```

Comment:
Cleanup ideas:
1. I'd make this into an array of objects!!
2. Rather than a `type: string`, I'd probably either a. use a `known` boolean or b. not have `type` at all and infer known based on whether label is a number